package.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "heroku-papertrail",
+  "version": "0.0.1",
+  "description": "Search and/or tail logs (optional: tail; search query)",
+  "main": "index.js",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/papertrail/papertrail-heroku-plugin"
+  },
+  "bugs": {
+    "url": "https://github.com/papertrail/papertrail-heroku-plugin/issues"
+  },
+  "keywords": [
+    "heroku-plugin",
+    "papertrail",
+    "log"
```

Comment:
I'd propose `logs`, `logging`, and `papertrailapp` as well. I just [checked](https://www.npmjs.com/search?q=logs) and substrings aren't automatically matched.